common/constants/runstate/runStates.go

```diff
@@ -5,16 +5,16 @@ import "fmt"
 type RunState int
 
 const (
-	New      RunState = 0 // State of a newly created Factomd object
-	Booting  RunState = 1 // State when starting up the server
-	Running  RunState = 2 // State when doing processing
-	Stopping RunState = 3 // State when shutdown has been called but not finished
-	Stopped  RunState = 4 // State when shutdown has been completed
+	New      RunState = iota // State of a newly created Factomd object
+	Booting  RunState = iota // State when starting up the server
+	Running  RunState = iota // State when doing processing
+	Stopping RunState = iota // State when shutdown has been called but not finished
+	Stopped  RunState = iota // State when shutdown has been completed
```

Comment:
You do not need to place iota on each line.

See: https://play.golang.org/p/1Ic9HaJwB5m

Reply:
Reviewed, agree with @Emyrk here. Rest looks good.

Reply:
Fixed and checked in. Thanks!